Dockerfile

```diff
@@ -138,13 +151,15 @@ ENV HZ_CLIENT_MODE=false \
 RUN  mkdir -p /opt/pega/kafkadata && \
      chgrp -R 0 /opt/pega/kafkadata && \
      chmod -R g+rw /opt/pega/kafkadata && \
+     chmod -R o+rwx /opt/pega/kafkadata && \
      chown -R pegauser /opt/pega/kafkadata
 
 # Set up dir for prometheus lib
 RUN mkdir -p /opt/pega/prometheus && \
     curl -sL -o /opt/pega/prometheus/jmx_prometheus_javaagent.jar https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.15.0/jmx_prometheus_javaagent-0.15.0.jar && \
     chgrp -R 0 /opt/pega/prometheus && \
```

Comment:
There's probably a good reason for all these seemingly redundant change group to root group commands, but just in case there isn't, I'm going to mention it again.

Reply:
same as other chgrp, seems redundant